artview/components/plot_grid.py

```diff
@@ -586,7 +586,7 @@ def cmapSelectCmd(self, cm_name):
 
     def toolZoomPanCmd(self):
         '''Creates and connects to a Zoom/Pan instance.'''
-        from .tools import ZoomPan
+        from .toolbox import ZoomPan
```

Comment:
The reason I did this was to avoid confusion with the tools dictionary that is used. I could revert back if you like.